Instrukcije_d001/src/app/terminal.h

```diff
@@ -0,0 +1,21 @@
+//
+// Created by Edin on 15. 03. 2022.
+//
+
+
+/* V H datotekah ne bomo imeli spremenljivk,
+ * te bodo normalno v C datotekah
+ * */
+#ifndef GITHUB_TERMINAL_H
+#define GITHUB_TERMINAL_H
+
+#include "World.h"
+#include <curses.h>
```

Comment:
Tukaj lahko tole foro narediš da narediš projekt cross-platform...

https://stackoverflow.com/questions/6649936/c-compiling-on-windows-and-linux-ifdef-switch